integration-tests/steps/stacks.py

```diff
@@ -442,6 +473,16 @@ def step_impl(context, a_or_no, kind):
         assert getattr(writer, difference_property) is test_value
 
 
+@then('stack drift status is "{desired_status}"')
+def step_impl(context, desired_status):
+    assert context.output[0]["StackDriftStatus"] == desired_status
+
+
+@then('stack resource drift status is "{desired_status}"')
+def step_impl(context, desired_status):
+    assert context.output[0][1]["StackResourceDrifts"][0]["StackResourceDriftStatus"] == desired_status
```

Comment:
Rather than scattering these steps across multiple already-bloated step files, I think you should create a new steps file for all the drift-related steps. Then you can have them all in one place and it would be easier to follow where things are coming from.